girder-dandi-archive/girder_dandi_archive/__init__.py

```diff
@@ -19,6 +22,9 @@ def load(self, info):
             {"$setOnInsert": {"value": 0}},
             upsert=True,
         )
+        Setting().set(SettingKey.CORS_ALLOW_ORIGIN, "*")
```

Comment:
Can you delete [this code line](https://github.com/dandi/dandiarchive/blob/166474feb825a1c75aae58ff278ca9d75fd48dc4/docker/provision/provision_girder.py#L51), since it is now obsolete?

Also, you can remove the entire [`setGirderSetting` function](https://github.com/dandi/dandiarchive/blob/166474feb825a1c75aae58ff278ca9d75fd48dc4/docker/provision/provision_girder.py#L10), since it's not used anywhere else.